.gitignore

```diff
@@ -12,6 +12,7 @@ Thumbs.db
 .lighthouseci
 .idea*
 .python-version
+_temp
```

Comment:
Similar to the earlier comments: I think every directory or file at root-level should have some relation to the actual 11ty setup, so I'd move this to tools/devtools/temp. This will also make it obvious in the .gitignore what it's for and won't draw confusion when working on things like https://github.com/GoogleChrome/developer.chrome.com/issues/65